docs/new_instrument.rst

```diff
@@ -81,19 +83,19 @@ When combined with the platform, this forms a unique file in the `instruments`
 directory.  Examples include the EUV instrument on ICON (icon_euv) and the
 Incoherent Scatter Radar at JRO (jro_isr).
 
+**tag**
+
+In general, the tag points to a specific data product.  This could be a
+specific processing level (such as L1, L2), or a product file (such as the
+different profile products for cosmic_gps data, 'ionprf', 'atmprf', ...).
+
```

Comment:
I tend to view the hierarchy as `inst_id` then `tag` as this is how we nest the dictionaries for `_test_dates`, etc.  But I'm open to discussion.

Reply:
That is how we treat it in the code, but that's not how it's treated with most of the data sets (especially non-satellite data sets) or in the Instrument input.  Because of this, I think the hierarchy should be kept as `platform`, `name`, `tag`, `inst_id`.  But if you would like to discuss this at the next meeting I think we should do that.